package.json

```diff
@@ -18,10 +18,11 @@
     }
   },
   "lint-staged": {
-    "*.{json,yaml,yml}": "prettier --write",
-    "*.md": [
-      "markdownlint --fix",
+    "*.{css,html,json,md,yaml,yml}": [
```

Comment:
add `,ts`

Reply:
I copied/merged these from the monorepo and would like to be consistent between them, so will update this to add js,ts,tsx and reflect the same extensions over in the monorepo root. Thanks for the review!

Reply:
oh question though, should we move the markdownlint step before the formatting step? and maybe it's better to format it like we do with the monorepo, e.g.

```markdown
  "lint-staged": {
    "*.{css,html,js,json,jsx,ts,tsx,yaml,yml}": [
      "prettier --write"
    ],
    "*.md": [
      "markdownlint --fix",
      "prettier --write"
    ]
  }
```

Reply:
yea, makes sense. I was trying to combine them to avoid repeating things but the existing approach is clearer